Floofbot/Services/WelcomeGateService.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Discord.WebSocket;
```

Comment:
Why the separate file? Looks like all the logic is handled in `EventHandlerService`

Reply:
![image](https://user-images.githubusercontent.com/35963613/104973441-99a7d880-59ec-11eb-9a3d-ef970f5c4763.png)